cache/test_cache_control.py

```diff
@@ -806,9 +809,53 @@ class HttpChainCacheActionCached(TestCacheControl, SingleTest):
             cookie "foo_items_in_cart" == "*" -> $$cache = 0;
             cookie "comment_author_*" == "*" -> $$cache = 1;
             cookie "wordpress_logged_in*" == "*" -> $$cache = 0;
-            -> vh1;
+            -> default;
```

Comment:
With #1650 fix `default` is a special vhost name again, so the change is likely invalid

Reply:
With #1650 resolved, I still don't see a reason for the change.